src/test/java/com/commercetools/testUtil/JsonAssertUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.commercetools.testUtil;
+
+import com.jayway.jsonpath.JsonPath;
+import org.hamcrest.Matcher;
+import org.springframework.test.util.JsonPathExpectationsHelper;
+
+public final class JsonAssertUtil {
+
+    /**
+     * Assert {@code content} string is JSON-formatted and contains {@code jsonPath} value matching {@code matcher}.
+     * <p>
+     * Example:<pre>
+     * assertJsonPath("{\"key\": \"value\"}", "$.key", is("value"));
+     * assertJsonPath("{\"arrays\":{\"arrayKey\": [1, 2, 3]}}", "$.arrays.arrayKey", contains(1, 2, 3));
+     * </pre>
+     *
+     * @param content  JSON string content
+     * @param jsonPath json path which value to assert.
+     *                 See more at {@link JsonPath#compile(java.lang.String, com.jayway.jsonpath.Predicate...)}
+     * @param matcher  matcher to apply to the value fetched from {@code jsonPath}
+     */
+    public static void assertJsonPath(String content, String jsonPath, Matcher<?> matcher) {
+        (new JsonPathExpectationsHelper(jsonPath)).assertValue(content, matcher);
+    }
+
+    private JsonAssertUtil() {
```

Comment:
Curious to know your opinion on having the private constructor throw an exception (e.g. throw new `UnsupportedOperationException` or `IllegalStateException`) which makes it more safer against potential instantiation from inside the util class?

Reply:
If someone wants to shoot himself in the foot - i can't protect completely.
My task is notify the users/customers/followers about potential danger. Otherwise we should go even deeper and protect agains instantiation over reflection and so on :)

Reply:
> for example, if an other developer can instantiate from inside the util class, nothing protects him to add a new constructor, isn't it?

Agreed.

As I said: "I was curious to know your opinion" 😉 , because I saw it somewhere and was wondering about it's significance and what do you think about it. I never done it myself for instance. 

I also don't know how smart the dev who would want to do that, and I actually don't tend to care about or judge. We are all learning 😉